ui/src/views/user/signup-thankyou/SignupThankyouView.vue

```diff
@@ -8,7 +8,7 @@
       <v-col cols="auto">
         <content-card 
           title="Thanks for registering!" 
-          description="Please check your inbox or spams  to confirm your registration" 
+          description="Please check your inbox or spams to confirm your registration" 
           :image="require('@assets/svg/man-woman-holding-mail.svg')"
         >
           <template #actions>
```

Comment:
For the button styling, you can do it with: 
```
            <v-btn text class="text-none pa-0">
           
```

This will reset the padding to 0 and remove the text-transform.

Reply:
done
<img width="960" alt="done" src="https://user-images.githubusercontent.com/64476430/172062991-957998c8-d2a7-414a-b23a-c3fb9dd5f31c.PNG">

Reply:
Thanks for the hard work! Approved